joplin/base/models/service_page.py

```diff
@@ -33,8 +33,12 @@ class ServicePage(JanisBasePage):
             )),
             ('step_with_options_accordian', StructBlock(
                 [
-                    ('options_description', TextBlock(
-                        'Describe the set of options')),
+                    ('options_description', RichTextBlock(
+                        features=WYSIWYG_SERVICE_STEP,
+                        # richTextPlaceholder.js searches for the class 'coa-option-description' and replaces placeholder text
```

Comment:
I would also add a note that the text (""Option description") is currently hardcoded by richTextPlaceholder.js (i like to leave comments on tech debt because I feel like it makes intentions more obvious to someone who might come along later)